src/extensions/extensions-store/extensions-store.injectable.ts

```diff
@@ -7,14 +7,7 @@ import { ExtensionsStore } from "./extensions-store";
 
 const extensionsStoreInjectable = getInjectable({
   id: "extensions-store",
-
-  instantiate: () => {
-    ExtensionsStore.resetInstance();
-
-    return ExtensionsStore.createInstance();
-  },
-
-  causesSideEffects: true,
```

Comment:
In wonder if there's still side-effects in `ExtensionStore`.

Reply:
Not anymore, or at least not by the time the PR is finished everything will be injected